addons/base_vat/base_vat.py

```diff
@@ -115,6 +115,9 @@ def button_check_vat(self, cr, uid, ids, context=None):
         return True
 
     def check_vat(self, cr, uid, ids, context=None):
+        context = context or {}
+        if context.get("openupgrade_disable_vat_validation"):
```

Comment:
We don't modify modules code in OpenUpgrade. I'm not sure if allowing this exception... @mreficent what do you think?

Reply:
We don't have clients at v6.1, so do as you like. But if you want this, at least add a comment saying it's an openupgrade fix.

Reply:
I think that when it's necessary it must be allowed. Otherwise, separating OpenUpgrade form OCB wouldn't make sense.

Reply:
Comment added.